.gitignore

```diff
@@ -35,6 +35,7 @@ tools/vms-generator/vms-generator
 .coverprofile*
 coverage.html
 manifests/**/*.tmp
+cluster-up/**/kind-k8s-sriov*/**/*.cert
```

Comment:
Shouldn't we solve this in kubevirtci rather?

Reply:
the files are created each run of cluster-up in kubevirt folder
and they appear on git status of kubevirt itself, so unless the mechanism would totally refactored, i think its good to go ?

Reply:
Sorry for being not explicit enough: I meant that if kubevirtci is creating these files and the cluster-up folder is vendored in we should just add a `.gitignore` ignoring the created files within kubevirtci, so that none of the users vendoring kubevirtci have this problem? Otherwise each of the users of kubevirtci would need to solve this on it's own, right?

Reply:
I mean, I'm not against fixing this here temporary, but we need to tackle this in kubevirtci.

Reply:
not kubevirtci creating those, but the running of `make cluster-up` under kubevirt folder
its created on the fly, so just a change of mechanism or deleting the files when the cluster-up finish would solve it otherwise,

we can add the same .gitignore at kubevirtci, but this one is for `make cluster-up` which is done under kubevirt
wdyt ?

Reply:
I can't follow? `make cluster-up` is calling [`./cluster-up/up.sh`](https://github.com/kubevirt/kubevirt/blob/72d0d9b8a51c59d921074ac003014b2a3e214a1a/Makefile#L114-L113) which is inside the folder that is vendored in from kubevirtci?

Reply:
So, like I said, kubevirtci itself is creating the files and therefore the .gitignore should be created in kubevirtci.

Reply:
its created by the scripts of cluster-up, by only when you run the command, and if you run it on kubevirt folder, 
you must either change the mechanism of kubevirtci to not create those or delete them, 
or to .gitignore them in kubevirt itself, its orthogonal to .gitignore of kubevirtci which might want to do the same thing for itself


those 2 cert files arent vendored, they even not under git control nowhere, but created by the cluster-up script suite.

Reply:
https://github.com/kubevirt/kubevirtci/pull/456
Thanks , see this one please, closing this one

Reply:
So you are saying that if the .gitignore file is created within cluster-up directory inside kubevirtci, then the .gitignore files inside cluster-up are not vendored in, but skipped?

Reply:
yes just tested it, touched a file and it works, opened a PR for it on kubevirtci

Reply:
Yes, I understand, the certs are created when running `cluster-up/up.sh` for the kind sriov provider.

All I'm saying is that in general if a _temporary_ file is created on the fly in a repository in a directory that is under version control (i.e not itself listed in a .gitignore), this file should be listed in a `.gitignore` in that directory in the repository to avoid confusion.